openstack/cinder/files/mitaka.cinder.conf

```diff
@@ -1,13 +1,12 @@
 {% from "virl.jinja" import virl with context %}
-
+{% from "openstack/worker_pool.jinja" import worker_count, db_pool %}
```

Comment:
Please also see the other comments I made, displayed in the conversation tab (or on the first commit).
Note that by default the OVA will be sized the same as the build packer instance, which is the smallest possible (machine_size = 0).
I also intend to create a new sls which only makes these pool changes, so that a controller can run them to adapt in size to the actual dimensions/cluster status it has.